modules/linear_boltzmann_solvers/b_discrete_ordinates_solver/sweep/sweep.h

```diff
@@ -19,10 +16,18 @@ enum class FaceOrientation : short
   OUTGOING = 1
 };
 
-struct STDG;     ///< Global Sweep Plane Ordering
-struct SPLS;     ///< Sweep Plane Local Subgrid
-class AAH_FLUDS; ///< Auxiliary Flux Data Structure
-class SPDS;      ///< Sweep Plane Data Structure
+enum class AngleSetStatus
+{
+  NOT_FINISHED = 0,
+  FINISHED = 1,
+  RECEIVING = 2,
+  READY_TO_EXECUTE = 3,
+  EXECUTE = 4,
+  NO_EXEC_IF_READY = 5,
+  MESSAGES_SENT = 6,
+  MESSAGES_PENDING = 7
+};
+typedef AngleSetStatus ExecutionPermission;
```

Comment:
I know this is just a moved piece of code, but interesting. I would not see right away why `Status` maps to `ExecutionPermission`. I am not requesting any action ATM, just pointing that out...

Reply:
Me either, and I've removed this in a future commit.

Reply:
Ok, good :+1: